subjects/rectperimeter/README.md

```diff
@@ -0,0 +1,32 @@
+# rectperimeter
+
+### Instructions
+
+write a function that accepts two integer arguments, representing the two sides length of a rectangle and returning the perimeter of the rectangle.
+
+your function should return -1 if one of the arguments is negative.
+
+### Expected function
+
+```go
+func RectPerimeter(l, w int) int {
+
+}
+```
+
+### Usage
+
```

Comment:
Missing "Here is a possible program to test your function:"

Reply:
fixed